src/app/components/grid/index.html

```diff
@@ -103,29 +160,281 @@
+					<!-- Activate the summary action bar link once page is added -->
+					<!-- <li><a routerLink="/components/summary-actionbar">Summary action bar</a></li> -->
+					<li><a routerLink="/components/infinite-scroll">Infinite scroll</a></li>
+					<li><a routerLink="/components/pagination">Pagination</a></li>
+					<li><a routerLink="/components/toolbar">Toolbar</a></li>
+					<li><a routerLink="/components/filter">Filter</a></li>
+					<li><a routerLink="/components/sort">Sort</a></li>
+				</ul>
+			</stache-column>		
+			<stache-column screenSmall="4" screenMedium="3">
+				<header><div class="sky-section-heading">Related guidelines</div></header>
+				<ul>
+					<li><a routerLink="/design/guidelines/page-layouts">Page layout</a></li>
+				</ul>
+			</stache-column>
+		</stache-row>
+		
+		
+	<header><div class="sky-subsection-heading">When to use grids</div></header>
+		
+  <stache-row>
+    <stache-column screenSmall="6">
+			<p>Use grids to display tabular data when: </p>
+			<ul>
+				<li>There is a large amount of data that needs to be visible at once.</li>
+				<li>Users need to visually scan a column to look for certain values or outliers.</li>
+				<li>Users need to compare values between rows.</li>
+			</ul>
+		</stache-column>
+		<stache-column screenSmall="6">
+			<div class="screenshot screenshot-success">
+				<img src="~/assets/img/guidelines/grid/grid-use-when.png" />
+			</div>
+		</stache-column>
+	</stache-row>
+    
+	<header><div class="sky-subsection-heading">When not to use grids</div></header>
+
+  <stache-row>
+    <stache-column screenSmall="6">
+			<ul>
+        <li>Do not use grids when you need to display visual content such as graphs or images. Use <a routerLink="/components/repeater">repeaters</a> or <a routerLink="/components/card">card</a> views instead.</li>
+        <li>Do not use grids when the content calls for a more complex layout. If a grid requires a lot of templated columns, try a repeater instead.</li>
+        <li>Do not use grids when you need to display many fields and it is likely that users will view the grids in a small viewport, such as on a phone. Try repeaters instead.</li>
+      </ul>
+		</stache-column>
+		<stache-column screenSmall="6">
+			<div class="screenshot screenshot-danger">
+				<img src="~/assets/img/guidelines/grid/grid-dont-use-complex.png" style="width: 100%;" />
+			</div>
+		</stache-column>
+	</stache-row>
+		
+	<header><div class="sky-section-heading">Anatomy</div></header>
+	
+		<stache-row>
+			<stache-column screenSmall="3">
+				<ol>
+					<li>
+						<span class="sky-deemphasized">List count (optional)</span>
+					</li>
+					<li>
+						<span class="sky-deemphasized">Toolbar (optional)</span>
+					</li>
+					<li>
+						Header row
+					</li>
+					<li>
+						<span class="sky-deemphasized">Row selection checkbox (optional)</span>
+					</li>
+					<li>
+						<span class="sky-deemphasized">Context menu button (optional)</span>
```

Comment:
I'm curious - "List count" and "Context menu button" aren't native functionality of the grid. The consumer will have to build them. Listing them here makes it seem like they're offered in this library. Is that ok? Or maybe this is more of a guideline of how to use those types of things if you want to build them yourself?

Reply:
Makes sense. That's kind of what I figured, but just wanted to mention it. I don't think it should hold up the PR. Just something to think about as we flesh these docs out.